### PR TITLE
Prac 7 code review request

### PR DESCRIPTION
**Pull request by StuBeattie**
Please check formatting, naming and logic.
@Matthew-Marsh

### CODE REVIEW

File: CP1404_practicals/prac_08/convert_miles_km.py [hunk 1]
```diff
@@ -7,12 +7,37 @@
...
+        """ build the Kivy app from the kv file """
+        Window.size = (600, 300)
+        self.title = "Convert Miles to Kilometers"
+        self.root = Builder.load_file('convert_miles_km.kv')
+        return self.root
+
+    def handle_calculate(self, value):
+        """ handle calculation (could be button press or other call), output result to label widget """
+        result = self.convert_value(value) * MILES_TO_KM_CONVERSION
+        self.root.ids.output_label.text = "{:,.3f}".format(result)
```
Comment (Matthew-Marsh) [inline CP1404_practicals/prac_08/convert_miles_km.py:26]: Could have used StringProperty for this part. But this wall will change the formatting, so it's understandable.

File: CP1404_practicals/prac_08/dynamic_labels.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    def create_widgets(self):
+        """Create labels from list entries and add them to the GUI."""
+        for name in self.names:
+            # create a label for each name in the list.
+            temp_label = Label(text=name)
+            # add the label to the "entries_box" layout widget
+            self.root.ids.entries_box.add_widget(temp_label)
+            # update status text
+            self.status_text = "{}".format(name)
```
Comment (Matthew-Marsh) [inline CP1404_practicals/prac_08/dynamic_labels.py:40]: This is unnecessary for this program. It was a label on the  Dynamic Buttons program that would display the phone number of the clicked button. You could also remove the StringProperty import and status_text = StringProperty() lines for the same reason.

**Review by Matthew-Marsh — COMMENTED** [review]
Hey Stuart,

Left a couple of comments, but I can't see anything too major that needs correcting.

Cheers,
Matt

File: CP1404_practicals/prac_08/dynamic_labels.py (hunk 2 above)
Comment (StuBeattie) [reply CP1404_practicals/prac_08/dynamic_labels.py:40]: Thanks for the input Matthew. My understanding of the dynamic GUI still isn't great so I will be going through it today and try sort it out.

**Pull request merged**